### ISSUE TITLE
question on KL Loss for training domain A

### ISSUE THREAD

**Issue by lulujianjie** [microsoft/Bringing-Old-Photos-Back-to-Life#170]
I see the code of KL loss as `loss_G_kl = torch.mean(torch.pow(hiddens, 2)) * self.opt.kl`

Will it limit the hidden tensors to get smaller?

**Comment by raywzy** [comment]
Great question! The VAE in domain A will search the balance between KL loss and the reconstruction error, so the hidden tensors would not be smaller all the time.
